<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HasUpdateWindow</class>
 <widget class="QMainWindow" name="HasUpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FoxyFace has Update</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="header_lb">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>A new update for FoxyFace has been found</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="version_lb">
      <property name="text">
       <string notr="true">FoxyFace update: 1.0.0.0 -&gt;1.0.0.0</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="description_lb">
      <property name="text">
       <string>Clicking on the &quot;Open Browser&quot; button will open a link in the browser from which you can download the latest version of the program.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="open_btn">
        <property name="text">
         <string>Open Browser</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ignore_update_btn">
        <property name="text">
         <string>Ignore Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="close_btn">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>open_btn</sender>
   <signal>clicked()</signal>
   <receiver>HasUpdateWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ignore_update_btn</sender>
   <signal>clicked()</signal>
   <receiver>HasUpdateWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>HasUpdateWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
